cpp/models/ode_secirvvs/parameter_space.cpp

```diff
@@ -67,8 +67,8 @@ void draw_sample_infection(Model& model)
     model.parameters.get<Seasonality>().draw_sample();
 
     //not age dependent
-    model.parameters.get<IncubationTime>()[AgeGroup(0)].draw_sample();
-    model.parameters.get<SerialInterval>()[AgeGroup(0)].draw_sample();
+    model.parameters.get<TimeExposed>()[AgeGroup(0)].draw_sample();
```

Comment:
Why can't TimeExposed and TimeInfectedNoSymptoms be age dependent? I would add them to the age dependent part where also the other stay times are located

Reply:
See comment above. Keep this for now because of backward stability. Will improve this in another issue soon.